lib/src/cli_client.dart

```diff
@@ -16,29 +16,27 @@ import 'services/travis.dart' as travis;
 
 class CommandLineClient {
   final String projectDirectory;
-  final String packageRoot;
```

Comment:
Wouldn't your implementation allow for support of both packages file and directory (XOR), same as the SDK does?